daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1230,10 +1251,12 @@ void IEEE1588Port::getDeviceTime
 
 void IEEE1588Port::becomeMaster( bool annc ) {
 	port_state = PTP_MASTER;
-	// Start announce receipt timeout timer
-	// Start sync receipt timeout timer
+	// Stop announce receipt timeout timer
 	clock->deleteEventTimer( this, ANNOUNCE_RECEIPT_TIMEOUT_EXPIRES );
```

Comment:
I agree with regards to the announce receipt timer and considered updating it but decided better to not change code I wasn't going to be in a position to test.